App/Properties/Settings.Designer.cs

```diff
@@ -94,5 +94,35 @@ internal sealed partial class Settings : global::System.Configuration.Applicatio
                 this["OsuDirectory"] = value;
             }
         }
+
+        [global::System.Configuration.UserScopedSettingAttribute()]
+        [global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
+        [global::System.Configuration.DefaultSettingValueAttribute("")]
+        public string DownloadManager_SaveDirectory
+        {
+            get
+            {
+                return ((string)(this["DownloadManager_SaveDirectory"]));
+            }
+            set
+            {
+                this["DownloadManager_SaveDirectory"] = value;
+            }
+        }
+
+        [global::System.Configuration.UserScopedSettingAttribute()]
+        [global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
+        [global::System.Configuration.DefaultSettingValueAttribute("")]
+        public string DownloadManager_AuthorizationCookies
+        {
+            get
+            {
+                return ((string)(this["DownloadManager_AuthorizationCookies"]));
+            }
+            set
+            {
+                this["DownloadManager_AuthorizationCookies"] = value;
+            }
+        }
```

Comment:
This was created by hand, instead of using a settings designer.
Adding or editing any setting in designer results in this code getting removed.

Reply:
Got it. My bad again, thank you for you attention.